.github/workflows/support-workers-build.yml

```diff
@@ -96,8 +99,7 @@ jobs:
                     - "-FailureHandlerLambda-"
                     - "-SendAcquisitionEventLambda-"
                     - "-PreparePaymentMethodForReuseLambda-"
-                  fileName: support-workers.jar
-                dependencies: [cfn]
+                  fileName: support-workers-${{ github.run_number }}.jar
```

Comment:
jar name has to change every time, otherwise lambda won't let us publish a new version.  I suspect if we change anything about the Lambda it would let us, perhaps even the description?